### PR TITLE
Filter favorites in schedule

### PR DESCRIPTION
**Pull request by eshiafr2**

### CODE REVIEW

File: app/src/main/java/org/hackillinois/android/view/schedule/FavEventFragment.kt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package org.hackillinois.android.view.schedule
+
+class FavEventFragment {
```
Comment (bstrauch24) [inline app/src/main/java/org/hackillinois/android/view/schedule/FavEventFragment.kt]: Delete this class if you're not using it!

File: app/src/main/java/org/hackillinois/android/view/schedule/ScheduleFragment.kt [hunk 2]
```diff
@@ -8,12 +8,15 @@ import androidx.fragment.app.FragmentPagerAdapter
...
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Button
 import kotlinx.android.synthetic.main.fragment_schedule.view.*
 import org.hackillinois.android.R
 import org.hackillinois.android.viewmodel.ScheduleViewModel
 
 class ScheduleFragment : Fragment() {
 
+    private lateinit var favoriteScheduleButton: Button
```
Comment (bstrauch24) [inline app/src/main/java/org/hackillinois/android/view/schedule/ScheduleFragment.kt]: I would just call this variable `favoriteButton`. You are in the `ScheduleFragment` class, so the original variable name is a little redundant.

File: app/src/main/java/org/hackillinois/android/view/schedule/ScheduleFragment.kt [hunk 3]
```diff
@@ -41,4 +48,10 @@ class ScheduleFragment : Fragment() {
         override fun getItem(position: Int) = DayFragment.newInstance(position)
         override fun getCount() = 3
     }
+
+    private val favScheduleClickListener = View.OnClickListener {
+        favoriteScheduleButton.isSelected = true
+
```
Comment (bstrauch24) [inline app/src/main/java/org/hackillinois/android/view/schedule/ScheduleFragment.kt]: Get rid of these extra newlines here

**Comment by patrickfeltes** [comment]
> I know I'm not supposed to commit commented code, but those lines were giving errors and I wasn't sure how to resolve it to fix the functionality. Lines 75 and 77 need proper statements.

No worries! Thanks for working on this. I'll see if I can fix those lines

**Pull request merged**